frame/balances/src/lib.rs

```diff
@@ -412,6 +412,14 @@ decl_storage! {
 					"the balance of any account should always be at least the existential deposit.",
 				)
 			}
+
+			// ensure no duplicates exist.
+			let mut endowed_accounts = config.balances.iter().map(|(x, _)| x).cloned().collect::<Vec<_>>();
```

Comment:
```suggestion
			let endowed_accounts = config.balances.iter().map(|(x, _)| x).cloned().collect::<std::collections::HashSet<_>>();
```

Reply:
Ahh this is very nice, but I think it borked the build Sir :P